src/sidebar/components/NotebookResultCount.js

```diff
@@ -38,26 +38,28 @@ function NotebookResultCount({
   const threadCount = rootThread.children.length;
 
   return (
-    <div className="hyp-u-layout-row NotebookResultCount">
+    <div className="flex gap-x-1 leading-none">
       {isLoading && <Spinner size="small" />}
       {!isLoading && (
-        <h2>
-          {!hasResults && <strong>No results</strong>}
+        <h2 className="font-bold">
+          {!hasResults && <span>No results</span>}
           {hasResults && isFiltered && (
-            <strong>
+            <span>
               {matchCount} {matchCount === 1 ? 'result' : 'results'}
-            </strong>
+            </span>
           )}
           {hasResults && !isFiltered && (
-            <strong>
+            <span>
               {threadCount} {threadCount === 1 ? 'thread' : 'threads'}
-            </strong>
+            </span>
           )}
         </h2>
       )}
-      {hasForcedVisible && <h3>(and {forcedVisibleCount} more)</h3>}
+      {hasForcedVisible && (
+        <h3 className="italic font-normal">(and {forcedVisibleCount} more)</h3>
```

Comment:
Hmm, I'm not sure about the use of h1, h2 and h3 headings within this header. The UI here doesn't map neatly to the classic use of heading levels within a document, in the sense that there isn't any content logically grouped under the h2 or h3 headings.